### PR TITLE
 feat(LERN-65): implemented assign Admin to school and get all admins that are not assigned to school endpoints

### PR DESCRIPTION
**Pull request by A1eeee**

### CODE REVIEW

File: backend/src/main/kotlin/ch/learntrack/backend/backoffice/school/SchoolResource.kt [hunk 1]
```diff
@@ -22,4 +23,10 @@ public class SchoolResource(private val schoolService: SchoolService) {
     @GetMapping(params = ["assignedUserId"])
     public fun getSchoolByUser(@RequestParam assignedUserId: UUID): List<SchoolDto> =
         schoolService.fetchSchoolsByUserId(assignedUserId)
+
+    @PutMapping("/assignUserToSchool")
```
Comment (deepspace1000) [inline backend/src/main/kotlin/ch/learntrack/backend/backoffice/school/SchoolResource.kt:27]: yeah we really need to redo this endpoint XD

File: backend/src/main/kotlin/ch/learntrack/backend/backoffice/user/UserResource.kt [hunk 2]
```diff
@@ -34,4 +33,8 @@ public class UserResource(private val userService: UserService) {
     @GetMapping("/getStudentCount")
     public fun getStudentCounts(@RequestParam schoolId: UUID): Int = userService.getUserCount(schoolId,
         UserRole.STUDENT)
+
+    @GetMapping("/getAllAdminUsersNotAssignedToSchool")
```
Comment (deepspace1000) [inline backend/src/main/kotlin/ch/learntrack/backend/backoffice/user/UserResource.kt:37]: Yes we really do....

File: backend/src/test/kotlin/ch/learntrack/backend/backoffice/school/SchoolIntegrationTest.kt [hunk 3]
```diff
@@ -274,4 +277,41 @@ class SchoolIntegrationTest: IntegrationTest() {
...
+                uriBuilder
+                    .path("$BACKOFFICE_ROOT_URL/school/assignUserToSchool")
+                    .queryParam("schoolId", ethSchoolId)
+                    .queryParam("userId", userAdminTemplateId)
+                    .build()
+            }
+            .setBasicAuthHeader(backendProperties)
+            .exchange()
+            .expectStatus()
+            .is5xxServerError
```
Comment (deepspace1000) [inline backend/src/test/kotlin/ch/learntrack/backend/backoffice/school/SchoolIntegrationTest.kt:315]: why 500? shouldn't it be 409 Conflict?

**Review by deepspace1000 — APPROVED** [review]

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 1  | ❗       |
    


*❗ custom/admin-landing-page*: 


| Before | After |
| --- | --- |
|  |  |

**Pull request merged**